testing/helpers/service-helpers.js

```diff
@@ -278,6 +279,7 @@ export class BrowserController {
    * @return {!Promise}
    */
   waitForElementBuild(selectorOrElement, timeout = 5000) {
+    // eslint-disable-next-line local/query-selector
```

Comment:
can we globally disable this rule for `testing/*`?

Reply:
Yeah, that makes sense